website/docs/history-and-hard-forks/history-vm-version.mdx

```diff
@@ -0,0 +1,212 @@
+2. 10 [MOPs](https://en.wikichip.org/wiki/macro-operation_fusion) added:
+
+| Opcode                                                      | Origin                       |
+| ----------------------------------------------------------- | ---------------------------- |
+| ADC [2](https://github.com/nervosnetwork/ckb-vm/issues/169) | add + sltu + add + sltu + or |
+| SBB                                                         | sub + sltu + sub + sltu + or |
+| WIDE_MUL                                                    | mulh + mul                   |
+| WIDE_MULU                                                   | mulhu + mul                  |
+| WIDE_MULSU                                                  | mulhsu + mul                 |
+| WIDE_DIV                                                    | div + rem                    |
+| WIDE_DIVU                                                   | divu + remu                  |
+| FAR_JUMP_REL                                                | auipc + jalr                 |
+| FAR_JUMP_ABS                                                | lui + jalr                   |
+| LD_SIGN_EXTENDED_32_CONSTANT                                | lui + addiw                  |
+
+### Syscalls
+
+No new added syscalls.
+
+### Cost Model
+
+- For all B instructions, 1 cycle will be consumed.
+- 10 [MOPs](https://en.wikichip.org/wiki/macro-operation_fusion) cycle cost:
+
+| Opcode                                                      | Origin                       | Cycles            |
+| ----------------------------------------------------------- | ---------------------------- | ----------------- |
+| ADC [2](https://github.com/nervosnetwork/ckb-vm/issues/169) | add + sltu + add + sltu + or | 1 + 0 + 0 + 0 + 0 |
+| SBB                                                         | sub + sltu + sub + sltu + or | 1 + 0 + 0 + 0 + 0 |
+| WIDE_MUL                                                    | mulh + mul                   | 5 + 0             |
+| WIDE_MULU                                                   | mulhu + mul                  | 5 + 0             |
+| WIDE_MULSU                                                  | mulhsu + mul                 | 5 + 0             |
+| WIDE_DIV                                                    | div + rem                    | 32 + 0            |
+| WIDE_DIVU                                                   | divu + remu                  | 32 + 0            |
+| FAR_JUMP_REL                                                | auipc + jalr                 | 0 + 3             |
+| FAR_JUMP_ABS                                                | lui + jalr                   | 0 + 3             |
+| LD_SIGN_EXTENDED_32_CONSTANT                                | lui + addiw                  | 1 + 0             |
+
+## VM 2
+
+**introduced from 2nd Hard Fork – CKB Edition Meepo (2024)**
```

Comment:
```suggestion
Introduced since [2nd Hard Fork – CKB Edition Meepo (2024)](/docs/history-and-hard-forks/ckb-hard-fork-history#2nd-hard-fork--ckb-edition-meepo-2024)
```